src/main/java/org/broadinstitute/consent/http/service/LibraryCardService.java

```diff
@@ -180,7 +192,14 @@ private LibraryCard processUserOnNewLC(LibraryCard card) throws Exception {
                 //throw error here, user is trying to associate incorrect userId with email
                 throw new ConsentConflictException();
             }
+            card.setUserName(user.getDisplayName());
         }
         return card;
     }
+
+    private Boolean checkIsAdmin(User user) {
+        return user.getRoles()
+            .stream()
+            .anyMatch(role -> role.getName().equalsIgnoreCase(UserRoles.ADMIN.getRoleName()));
+    }
```

Comment:
This method also exists in `InstitutionUtil` and similar methods in `DacService`. We should start collecting those into a user utility.